editor/import/scene_import_settings.h

```diff
@@ -75,6 +76,9 @@ class SceneImportSettings : public ConfirmationDialog {
 
 	Camera3D *camera = nullptr;
 	Ref<CameraAttributesPractical> camera_attributes;
+	Ref<Environment> environment;
```

Comment:
Do we need those and the sky_material.h include really in the header? Those refs should be all kept by the camera anyway.